src/.eslintrc.js

```diff
@@ -40,6 +40,7 @@ module.exports = {
         '@typescript-eslint/no-unused-vars': [ 'error', { args: 'none' } ],
         'no-prototype-builtins': 'off',
         'prefer-rest-params': 'off',
-        'max-len': [ 'error', 130, 4 ]
+        'max-len': [ 'error', 130, 4 ],
+        'quotes': ['error', 'single'],
```

Comment:
note that linter config in src/ does not extend root conf,g file. If we did that, we would have to change a lot in src/ folder.  We could add some of them like we added max-len and quotes above. Let me know if we will add any other rules here as well.

Reply:
Yes, that's fine. Our rules for TS code are slightly different from JS ones.